utils/logger.js

```diff
@@ -0,0 +1,24 @@
+const pino = require('pino');
+// eslint-disable-next-line no-unused-vars
+const config = require('../src/config');
```

Comment:
`require('../src/config')` with no variable, and get rid of the eslint override.